scripts/deploy.sh

```diff
@@ -5,6 +5,7 @@ mv prometheus.yml prometheus/
 
 mv prod-docker-compose.yml docker-compose.yml
 
+#/home/ubuntu/deployment/scripts/migrate_all_sll_cert.sh
```

Comment:
When we have autogeneration of all certificates we can run this script on every deploy I guess (unless openssl changes the certificates somehow but I don't think so).

Reply:
That's what I mean, I commented it due to the fact that I accidentally deleted these certificates :(, so when they come back, this line must to be uncommented

Reply:
ok, seems legit